spec/unit/middleware/timeout_spec.rb

```diff
@@ -0,0 +1,177 @@
+require 'spec_helper'
+require 'qless/job'
+require 'qless/middleware/timeout'
+
+module Qless
+  module Middleware
+    ::Rspec.describe Timeout do
+      class JobClass
+        attr_accessor :perform
```

Comment:
Is this `perform` used anywhere?  As far as I can see, it's not.  Anyhow, Qless expects the `perform` method to be a class method on the job class, but here it is an instance method.  I guess it does not matter since this is in isolation as a unit test so it's not qless calling it but it is a bit confusing that it does not conform to the Qless job protocol but appears to be trying to.